Libraries/Image/RCTImageLoader.m

```diff
@@ -74,8 +74,7 @@ + (void)loadImageWithTag:(NSString *)imageTag callback:(void (^)(NSError *error,
           // doesn't spike the memory up during the process.
           @autoreleasepool {
             ALAssetRepresentation *representation = [asset defaultRepresentation];
-            ALAssetOrientation orientation = [representation orientation];
```

Comment:
@ide I am saying that this doesn't work for all images either, I am still looking for a solution to this problem. I am not sure what is actually happening.